featurebyte/models/observation_table.py

```diff
@@ -28,8 +34,38 @@ class SourceTableObservationInput(SourceTableRequestInput):
     """
 
 
+class TargetInput(FeatureByteBaseModel):
+    """
+    TargetInput is an input from a target that can eb used to create an ObservationTableModel
+    """
+
+    target_id: PydanticObjectId
+    observation_table_id: Optional[PydanticObjectId]
+    type: Literal[RequestInputType.OBSERVATION_TABLE, RequestInputType.DATAFRAME]
+
+    async def materialize(
+        self,
+        session: BaseSession,
+        destination: TableDetails,
+        sample_rows: Optional[int],
+    ) -> None:
+        """
+        No-op materialize.
```

Comment:
Would be good to add a comment on why this is a no-op. Does this function actually get called for `TargetInput`?

Reply:
good point - will add a comment and have it throw an exception, it shouldn't get called, but the linter throws an error since it expects it to be there!